.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!-- (an executive summary of the changes, ideally in one sentence) -->
+This PR...
+
+## Changes
+<!-- (major technical changes list) -->
+
+-
+
+## Checklist
+
+- [ ] tested with Python3.6
+- [ ] run `make check` or `make fix` for the formatting check
+- [ ] signed commits
+- [ ] updated documentation (e.g. parameter description, inline doc, docs.nitrokey)
+- [ ] added labels
```

Comment:
I think this should be more specific.  Added labels to this PR?

Reply:
Yes, I thought here about the PR and ticket labels. We can discuss further, how to make it best.